### DISCUSSION TITLE
OSCS 开源软件供应链安全社区 分享

### DISCUSSION THREAD

**Discussion by TieWay59** [X-lab2017/open-wonderland#88] [1️⃣ 新鲜事]
https://github.com/orgs/X-lab2017/discussions/81#discussioncomment-2895855

我准备好了一个几分钟的快捷分享。

我计划今天也就是周四晚上组会的议程后面接入我这个话题，不知道可不可行？ 

@will-ww @gymgym1212

**Comment by bifenglin** [comment]
好的，今晚（6月16日）会议是你的舞台。

**Comment by gymgym1212** [comment]
期待~

**Comment by zhicheng-ning** [comment]
![image](https://user-images.githubusercontent.com/39022409/174071822-cc0650bd-355b-41df-b4d3-4ea275a820a0.png)
